docs/linux-only-image.md

```diff
@@ -262,6 +262,149 @@ You should see the bootloader prompting you to start `openSUSE Tumbleweed`.
         elemental3-toolkit version
         ```
 
+## Create an Installer ISO
+
+In order to create a self installer ISO some configuration assets must be prepared in advance to include them into the ISO. The essential parts are:
+
+1. A configuration script
+2. A directory tree to include additional files into the ISO filesystem
+
+### Configure the Live Installer
+
+The ISO supports configurations through a script which will be executed in late initramfs just before switching root.
+The script is executed chrooting to the system root.
+
+#### Example Live Config Script
+
+In this example we are going to setup a configuration script that will set three aspects:
+
+1. Autologin, so the live ISO does not require a root password
+2. Add an elemental-autoinstaller service to run the installation at boot
+3. Ensure the extensions added in the ISO filesystem are linked to `/run/extesions` so they are loaded at boot
```

Comment:
Seems I cannot make suggestions here, but there's a small typo here: `/run/extensions` note the missing "n".